### PR TITLE
SCCM licesnse info added

### PR DESCRIPTION
**Pull request by joinimran**
SCCM license info was not added when it comes to Microsoft 365 EMS license. I have added the required information to answer user queries related to SCCM when they buy an EMS/Microsoft 365 license. 

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/332

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy edit - no changes required.

**Comment by mypil** [comment]
@jasongroce - Can you please assist to check, approve and merge the content for this PR based on issue #332?

Thank you.

**Comment by jasongroce** [comment]
Hi @mypil I'm no longer associated with this content. @jdeckerms can you help redirect this request?

**Comment by mypil** [comment]
@jasongroce - do you happen to know who is the new author for this article? please advise so we can update the metadata. Thank you.

**Comment by mypil** [comment]
@robmazz @denisebmsft - Do you happen to know who we can reach out to merge the content of this PR based on issue #332? Also, would you know who is the new author for this article? Thank you.

File: microsoft-365/business/support/microsoft-365-business-faqs.md [hunk 1]
```diff
@@ -112,7 +112,8 @@ Customers that wish to experience the complete capabilities of Microsoft 365 Bus
...
 
 ### Does Microsoft 365 Business include the full capabilities of Microsoft Intune? 
-Yes, Microsoft 365 Business subscribers are licensed to use full Intune capabilities for iOS, Android, MacOS, and other cross-platform device management. Features not available in the simplified management console in Microsoft 365 Business, like 3rd party app management and configuration of things like WiFi profiles, VPN certificates, etc., can be managed in full Intune console.   
+Yes, Microsoft 365 Business subscribers are licensed to use full Intune capabilities for iOS, Android, MacOS, and other cross-platform device management. Features not available in the simplified management console in Microsoft 365 Business, like 3rd party app management and configuration of things like WiFi profiles, VPN certificates, etc., can be managed in full Intune console.
```
Comment (nenonix) [inline microsoft-365/business/support/microsoft-365-business-faqs.md]:
```suggestion
Yes, Microsoft 365 Business subscribers are licensed to use full Intune capabilities for iOS, Android, MacOS, and other cross-platform device management. Features not available in the simplified management console in Microsoft 365 Business, like 3rd party app management and configuration of WiFi profiles, VPN certificates, etc, can be managed in full Intune console. 
```

**Review by nenonix — COMMENTED** [review]
change suggested

**Comment by aczechowski** [comment]
I happened to see mention if this change on Twitter. I'm the lead writer for SCCMDocs. Could you please remove the reference to the KB article, and instead refer to our FAQ on this topic? https://docs.microsoft.com/sccm/core/understand/product-and-licensing-faq#bkmk_csp
In case something changes in the future, I dont want to have to remember to track down this information elsewhere.

**Comment by mypil** [comment]
@aczechowski - are you the new article author taking over this article? Please advise so we can update the metadata information. Thank you.

**Comment by mypil** [comment]
@joinimran - please check on the recommendation from @aczechowski  and make the necessary changes to this PR. Thank you.

**Comment by aczechowski** [comment]
@mypil no sorry. Just hoping that @joinimran can revise it

**Comment by joinimran** [comment]
@aczechowski I have linked this part to SCCM Faqs. 

@mypil FYI Please.

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR or do you happen to know who we can reach out to review and merge the content?

Thank you!

**Comment by robmazz** [comment]
Added Sirkku per update on separate thread about ownership.

**Comment by mypil** [comment]
@Sirkkuw - Can you please assist to check, approve and merge the content for this PR based on issue #332?

Thank you.

**Comment by mypil** [comment]
@Sirkkuw - I would like to follow up. Can you please assist to check, approve and merge the content for this PR based on issue #332?

Thank you.

**Comment by mypil** [comment]
@Sirkkuw - Are you the new doc owner of this article? Please confirm so we can update the author metadata of this article. Thank you.

**Comment by Sirkkuw** [comment]
@mypil  I am the author. For some reason @jasongroce added himself in the author field after he suggested edits. Those fields are meant for the writers in the authoring group only.

**Pull request merged**